src/metpy/calc/thermo.py

```diff
@@ -1166,6 +1167,15 @@ def _check_pressure(pressure):
     return np.all(pressure[:-1] >= pressure[1:])
 
 
+def _check_pressure_error(pressure):
+    """Raise an `InvalidSoundingError` if _check_pressure returns False."""
+    if not _check_pressure(pressure):
+        msg = """
+        Pressure increases between at least two points in your sounding.
+        Using scipy.signal.medfilt may fix this."""
+        raise InvalidSoundingError(msg)
```

Comment:
I see this matches the original, but how about we just avoid `msg` here:
```suggestion
        raise InvalidSoundingError('Pressure increases between at least two points in '
                                   'your sounding. Using scipy.signal.medfilt may fix this.')
```
(or whatever wraps properly.)